traxxas_slash_4x4_platinum_2012/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#  Copyright (C) 2018-2021 LEIDOS.
```

Comment:
Copyrights should reflect the year of file creation, so 2021. Double check all files

Reply:
Does this include files copied verbatim from other configs? I've put the files I created under a FHWA copyright.

Reply:
Hmm, I suppose if no mods were made then it is no different from forking some else's repo so the Copyright should be left as is.

Reply:
I check through the configuration directory, and all files copied verbatim either retained their existing copyright notice or did not have one to begin with. All files I added or modified we put under an FHWA copyright.